azurerm/resource_arm_storage_blob_test.go

```diff
@@ -666,3 +711,45 @@ resource "azurerm_storage_blob" "destination" {
 }
 `, rInt, location, rString, sourceBlobName)
 }
+
+func testAccAzureRMStorageBlobPage_blockContentType(rInt int, rString, sourceBlobName, location, contentType string) string {
```

Comment:
_minor_ we've got a _vague_ convention here of making these `(rInt, rString, location, [others])`